### DISCUSSION TITLE
Friendly URL break when used in Multistore context with LiteSpeed server

### DISCUSSION THREAD

**Discussion by LameuleFR** [PrestaShop/PrestaShop#33245] [Q&A]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

After switching to LiteSpeed to replace Apache, the URL friendly rewrite no longer works on virtual multishops on the same domain.

According to my research, the .htaccess file is invalid and LiteSpeed server interprets it differently..

Friendly URl work normally only if u use a real secondary domain for the multishop.

### Expected behavior

Functional url rewriting like Apache or Adaptative htaccess to fit LiteSpeed server when using virtual shop on the same domain.

### Steps to reproduce

Install a Prestashop on a LiteSpeed server
Create a second shop with virtual url on the same domain
Category url & image show 404 error.

### PrestaShop version(s) where the bug happened

1.6.1.11 - 8.0

### PHP version(s) where the bug happened

All version. The problem is not related to PHP.

### Your company or customer's name goes here (if applicable).

AEROGRAPHEDISCOUNT

**Comment by hibatallahAouadni** [comment]
Hello @LameuleFR 

It seems your issue is a topic to discuss so @kpodemski could you please forward this issue to Discussion tab :pray: 
Thanks in advance!

**Comment by LameuleFR** [comment]
Hi @hibatallahAouadni 

I don't understand why the issue should switch to discussion, the bug is factual and easy to reproduce but whatever.

If you're wondering about the importance of LiteSpeed, it performs better than Apache or Nginx using their native caching system (LSCache). I get an average TTFB of 50ms, which is better than varnish in most cases.

LiteSpeed is supposed to be able to read the htaccess without any modification, the only problem is that it is much more verbose and strict than Apache. On my side, i've already tried to ask LiteSpeed about this problem, but so far I haven't had an answer.

**Comment by LameuleFR** [comment]
Fixed with recent update from LiteSpeed.
